parastell.py

```diff
@@ -76,10 +83,86 @@ def cubit_export(components, export, magnets):
             For a rectangular cross-section, the list format is
             ['rectangle' (str), width (float, cm), thickness (float, cm)]
     """
-    # Extract Cubit export parameters
-    facet_tol = export['facet_tol']
-    len_tol = export['len_tol']
-    norm_tol = export['norm_tol']
+    def legacy_export():
```

Comment:
I assume that because this function is defined inside the scope of the export function, that it doesn't need to pass all the variables....?

Reply:
yeah, local functions share the variables of the parent in a similar manner to how functions can access global variables, at least thats how I understand it